frame/display_server.py

```diff
@@ -0,0 +1,73 @@
+import inotify.adapters
+import os
+import time
+
+from program import Program, Command
+
+display_appear_timeout = 10
+min_frame_run_time = 0.1
+
+def clear_wayland_display(runtime_dir: str, name: str) -> None:
+    # Clear out any existing display before waiting for the new one
+    for path in [
+        os.path.join(runtime_dir, name),
+        os.path.join(runtime_dir, name + '.lock')
+    ]:
+        if os.path.exists(path):
+            os.remove(path)
```

Comment:
Changed the display name to use the process's PID, so it's not likely something else will be actively using the same display. I like clearing the display if it exists because this way there's no issue if a previous run went bad and didn't clean up properly, though I guess erroring might be a good idea instead.